CounterStrike/src/dev/danablend/counterstrike/utils/PacketUtils.java

```diff
@@ -2,107 +2,109 @@
+     * @param player
+     * @param titleText
+     * @param subtitleText
+     * @param fadeInDuration  - in seconds
+     * @param duration        - in seconds
+     * @param fadeOutDuration - in seconds
+     */
+    public static void sendTitleAndSubtitle(Player player, String titleText, String subtitleText, int fadeInDuration, int duration, int fadeOutDuration) {
+        player.sendTitle(titleText, subtitleText, 20 * fadeInDuration, 20 * duration, 20 * fadeOutDuration);
+    }
+
+    /**
+     * @param titleText
+     * @param subtitleText
+     * @param fadeInDuration  - in seconds
+     * @param duration        - in seconds
+     * @param fadeOutDuration - in seconds
+     */
+
+    public static void sendTitleAndSubtitleToInGame(String titleText, String subtitleText, int fadeInDuration, int duration, int fadeOutDuration) {
+        for (CSPlayer csplayer : CounterStrike.i.getCSPlayers()) {
+            sendTitleAndSubtitle(csplayer.getPlayer(), titleText, subtitleText, fadeInDuration, duration, fadeOutDuration);
+        }
+    }
+
+    public static void sendTitleAndSubtitleToWaitingInLobby(String titleText, String subtitleText, int fadeInDuration, int duration, int fadeOutDuration) {
+        for (Player player : Bukkit.getOnlinePlayers()) {
+
+            CSPlayer csplayer = CounterStrike.i.getCSPlayer(player, false, null);
+
+            if (csplayer == null && InLobbyLocation(player)) {
+                sendTitleAndSubtitle(player, titleText, subtitleText, fadeInDuration, duration, fadeOutDuration);
+            }
+        }
+    }
+
+    public static void sendTitleAndSubtitleToAll(String titleText, String subtitleText, int fadeInDuration, int duration, int fadeOutDuration) {
+        for (Player player : Bukkit.getOnlinePlayers()) {
+            sendTitleAndSubtitle(player, titleText, subtitleText, fadeInDuration, duration, fadeOutDuration);
+        }
+    }
+
+
+    public static void sendActionBarToInGame(String text) {
+        for (CSPlayer csplayer : CounterStrike.i.getCSPlayers()) {
+            sendActionBar(csplayer.getPlayer(), text);
+        }
+    }
+
+    public static void sendActionBarToWaitingInLobby(String text) {
+        for (Player player : Bukkit.getOnlinePlayers()) {
+
+            CSPlayer csplayer = CounterStrike.i.getCSPlayer(player, false, null);
+
+            if (csplayer == null && InLobbyLocation(player)) {
+                sendActionBar(player, text);
+            }
+        }
+    }
+
+    public static void sendActionBarToAll(String text) {
+        for (Player player : Bukkit.getOnlinePlayers()) {
+            sendActionBar(player, text);
+        }
+    }
+
+    public static void sendActionBar(Player player, String text1) {
+        final TextComponent component = Component.text(text1);
+
+        player.sendActionBar(component);
+
+        //player.sendMessage(ChatMessageType.ACTION_BAR, TextComponent..fromLegacyText(text));
+    }
+
+
+    private static Boolean InLobbyLocation(Player p) {

```

Comment:
function name should start with lowercase letter
```suggestion
    private static Boolean isInLobbyLocation(Player p) {

```